docs/quick_start.md

```diff
@@ -104,6 +104,12 @@ generator = Sublayer::Generators::CodeFromDescriptionGenerator.new(description:
 puts generator.generate
 ```
 
+### Step 5 - Explore Advanced Customization
```

Comment:
again this is the kind of change that i don't like or dislike 
i am happy to see the usage of the link however